File: leetcode/1662. Check If Two String Arrays are Equivalent.java
class Solution {
    public boolean arrayStringsAreEqual(String[] word1, String[] word2) {
        StringBuffer sb1 = new StringBuffer();
        StringBuffer sb2 = new StringBuffer();

        for (String e : word1)
            sb1.append(e);
        for (String e : word2)
            sb2.append(e);

        return sb1.toString().equals(sb2.toString());
    }
}

File: leetcode/1662. Check If Two String Arrays are Equivalent.cpp
class Solution
{
public:
    bool arrayStringsAreEqual(vector<string> &word1, vector<string> &word2)
    {
        string a = "", b = "";
        for (string e : word1)
            a += e;
        for (string e : word2)
            b += e;
        return a == b;
    }
};